src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -157,4 +157,15 @@ To create a branch on the docs-website repo:
 3. Click on **New Branch**
 4. You will be prompted to name your new branch. Descriptive names are best. It's a great way to quickly clue people in to what your work is all about. For example, if you are working on What’s New pages, you might name the branch **Whats-new-updates**.
 
-When you create a new branch, don't forget to add the Jira ticket's key (DOC-1234) to the branch name and the PR title. 
\ No newline at end of file
+When you create a new branch, don't forget to add the Jira issue ID (DOC-1234) to the branch name and the PR title.
+
+## A note about GitHub permissions [#permissions]
+
+Writers on the Docs team work in branches, but contributors outside of the Docs team use forks. This also means that non-Docs contributors can't edit content on a branch directly, which can make collaboration tricky.
+
+GitHub permissions are used to determine who can work on a branch and who can work on a fork. On GitHub, access is determined by team and member roles. Although anyone can contribute to our public repository, additional access is granted through `Write` and `Admin` roles. (There are other defined roles that aren't often used.)
+
+Every member of the Docs team is granted the `Admin` role, which grants full access to the repository. Other New relic contributors are given the `Write` role, which permits assigning users and opening and closing pull requests.
```

Comment:
@paperclypse, how might I use these permission as a writer? Should I be changing them?

Reply:
That's a conversation we should certainly have! But it's tough to have that conversation if people don't understand what the current set up is.